cmd/logger.go

```diff
@@ -31,12 +31,28 @@ func NewLogger(verbose bool) *zap.Logger {
 		zap.AddCaller(),
 		zap.AddCallerSkip(1),
 	)
-	zapAtom.SetLevel(zapcore.InfoLevel)
 
-	if verbose {
-		zapAtom.SetLevel(zapcore.DebugLevel)
+	// set zap log level based on logLevel input
+	var zapLogLevel zapcore.Level
```

Comment:
if you are going this route then pass make the arg of `NewLogger` a `zapcore.Level`

```
func NewLogger(logLevel zapcore.Level) *zap.Logger {}
```

Reply:
Good point. done.